config/config.go

```diff
@@ -56,6 +57,8 @@ type ServiceConfig struct {
 	Host []string `mapstructure:"host"`
 	// port to bind the lura service
 	Port int `mapstructure:"port"`
+	// address to listen
+	Address string `mapstructure:"listen_address"`
```

Comment:
why not just `mapstructure:"address"` ?

Reply:
This field accepts IPv4 or IPv6 addresses only like `192.0.2.1` (IPv4), `2001:db8::68` (IPv6), or `::ffff:192.0.2.1` (IPv4-mapped IPv6). I think "address" might confuse users and try to set hostnames or ports. I would go for `listen_ip` instead.